front/app/containers/Admin/projects/project/index.tsx

```diff
@@ -156,7 +157,7 @@ export const AdminProjectsProjectIndex = ({
           <PhaseHeader phase={selectedPhase} tabs={getTabs(project.id)} />
         )}
 
-        <Box p="40px" background={colors.white}>
+        <Box p={`${defaultAdminCardPadding}px`} background={colors.white}>
```

Comment:
Any reason why `defaultAdminCardPadding` couldn't be defined with 'px' in the variable? Assume, it will always be used as px